stock_landed_segmentation/views/stock_landed_costs_view.xml

```diff
@@ -26,6 +26,7 @@
                     <field
                         name="move_ids"
                         widget="many2many_tags"
+                        domain="['|', ('unbuild_id', '!=', False), ('production_id', '!=', False), ('state', 'in', ('done',))]"
```

Comment:
I'd leave the domain in the model. That way, it applies in all circunstances and all views.

Reply:
I decided to leave it in the view this allowing to be flexible and quick to react